django/urls/resolvers.py

```diff
@@ -163,7 +163,7 @@ def __init__(self, regex, urlconf_name, default_kwargs=None, app_name=None, name
         # set of dotted paths to all functions and classes that are used in
         # urlpatterns
         self._callback_strs = set()
-        self._populated = False
+        self._populated = self._populating = False
```

Comment:
I guess I'm not strongly -1 but multiple variable assignment on one line is an uncommon style that I'm not sure we want to encourage.